modules/ROOT/pages/5.5_Financial_ratios.adoc

```diff
@@ -36,7 +36,7 @@ _Figure 135. 'Financial ratio' EO mock-up_
 REQUIREMENT(s) specified by the buyers can be place outside a group of QUESTION(s) (see any other previous criteria) or *inside* a group of *QUESTION(s)*, which is the case for financial ratios, as you can see in the data structure for this criterion, below.
 ====
 
-Notice how the spreadsheet has been used to specify the three different financial ratios of the above mock-up example: the XML example below was produced using a XSL-T transformation link:{url-blob}/dist/xslt/[style-sheet] that takes the link:{url-blob}/dist/cl/ods/[ODS spreadsheet] and produced the link:{url-blob}/dist/xml/[XML instance].
+Notice how the spreadsheet has been used to specify the three different financial ratios of the above mock-up example: the XML example below was produced using a XSL-T transformation link:{url-tree-tmp}/xml-examples/__xslt__/ods-data-structures-to-ESPD-xml/ESPDRequest-Annotated.xslt[ESPDRequest-Annotated] that takes the link:{url-tree-tmp}/xml-examples/ESPD-criterion.xml[ESPD-criterion] and produced the link:{url-tree-tmp}/xml-examples/ESPD-Request.xml[ESPD-Request].
```

Comment:
ESPDRequest-Annotated link does not work

Reply:
Scaped the underscore chars. Fixed.